modules/kafka/options.go

```diff
@@ -0,0 +1,79 @@
+package kafka
+
+import (
+	"context"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+type options struct {
+	// Listeners is a list of custom listeners that can be provided to access the
+	// containers form within docker networks
+	Listeners []KafkaListener
+}
+
+func defaultOptions() options {
+	return options{
+		Listeners: make([]KafkaListener, 0),
+	}
+}
+
+// Compiler check to ensure that Option implements the testcontainers.ContainerCustomizer interface.
+var _ testcontainers.ContainerCustomizer = (*Option)(nil)
+
+// Option is an option for the Redpanda container.
+type Option func(*options)
+
+// Customize is a NOOP. It's defined to satisfy the testcontainers.ContainerCustomizer interface.
+func (o Option) Customize(*testcontainers.GenericContainerRequest) error {
+	// NOOP to satisfy interface.
+	return nil
+}
+
+func WithClusterID(clusterID string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) error {
+		req.Env["CLUSTER_ID"] = clusterID
+		return nil
+	}
+}
+
+// WithListener adds a custom listener to the Redpanda containers. Listener
+// will be aliases to all networks, so they can be accessed from within docker
+// networks. At leas one network must be attached to the container, if not an
+// error will be thrown when starting the container.
+func WithListener(listeners []KafkaListener) Option {
```

Comment:
Let's document these options in the `./docs/modules/kafka.md` file too.

Reply:
Done